domains/alx.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "michaelradu",
+        "email": "miihairadu@gmail.com"
+    },
+    "record": {
+        "CNAME": "apex-loadbalancer.netlify.com"
```

1. I cannot reach your website
2. We do not support Netlify/Vercel, so if you want to use a Netlify/Vercel website you will need to use a redirect instead
```suggestion
        "URL": "https://apex-loadbalancer.netlify.app"
```